src/main/java/meteordevelopment/meteorclient/systems/modules/misc/ServerSpoof.java

```diff
@@ -73,14 +79,34 @@ public ServerSpoof() {
     }
 
     private class Listener {
+    	
+    	@EventHandler
+    	private void onClientBrandRetrive(ClientBrandRetrieverEvent event) {
+    		if (spoofBrand.get()) event.info.setReturnValue(brand.get());
+    	}
+    	
         @EventHandler
         private void onPacketSend(PacketEvent.Send event) {
             if (!isActive()) return;
             if (!(event.packet instanceof CustomPayloadC2SPacket)) return;
             Identifier id = ((CustomPayloadC2SPacket) event.packet).payload().id();
+            
+            if(spoofBrand.get()) {
+            	if (id.equals(BrandCustomPayload.ID)) {
+                    event.packet.write(new PacketByteBuf(Unpooled.buffer()).writeString(brand.get()));
+                } else if (StringUtils.containsIgnoreCase(id.toString(), "fabric:registry/sync")) {
+	                event.cancel();
+	                return;
+	            } else if (StringUtils.containsIgnoreCase(id.toString(), "fabric:container/open")) {
+	                event.cancel();
+	                return;
+	            } else if (StringUtils.containsIgnoreCase(id.toString(), "fabric-screen-handler-api-v1:open_screen")) {
+	                event.cancel();
+	                return;
+	            }
+            }
 
-            if (spoofBrand.get() && id.equals(BrandCustomPayload.ID))
-                event.packet.write(new PacketByteBuf(Unpooled.buffer()).writeString(brand.get()));
+            
 
             if (blockChannels.get()) {
                 for (String channel : channels.get()) {
```

Comment:
Just defaulting `channels` and set it to `fabric:registry/sync` `fabric:container/open` `fabric-screen-handler-api-v1:open_screen`